cycle-scripts-es-browserify/package.json

```diff
@@ -27,6 +28,7 @@
     "chalk": "^1.1.3",
     "cross-spawn": "^4.0.2",
     "dotenv": "^2.0.0",
+    "ejs": "^2.5.2",
```

Comment:
I didn't know how else to add it. I didn't quite understand your nested projects infrastructure. Maybe you could document how to add the tools to do the work?
I've never seen anything quite like it. Loads of duplication to maintain in this way...
I always just have a single set of project templates... I developed dozens of similar generators in the past and never had to resort to this.